releases/release-1.19/release-notes-draft.md

```diff
@@ -494,7 +502,7 @@
 - Kubeadm now forwards the IPv6DualStack feature gate using the kubelet component config, instead of the kubelet command line ([#90840](https://github.com/kubernetes/kubernetes/pull/90840), [@rosti](https://github.com/rosti)) [SIG Cluster Lifecycle]
 - Kubeadm: do not use a DaemonSet for the pre-pull of control-plane images during "kubeadm upgrade apply". Individual node upgrades now pull the required images using a preflight check. The flag "--image-pull-timeout" for "kubeadm upgrade apply" is now deprecated and will be removed in a future release following a GA deprecation policy. ([#90788](https://github.com/kubernetes/kubernetes/pull/90788), [@xlgao-zju](https://github.com/xlgao-zju)) [SIG Cluster Lifecycle]
 - Kubeadm: use two separate checks on /livez and /readyz for the kube-apiserver static Pod instead of using /healthz ([#90970](https://github.com/kubernetes/kubernetes/pull/90970), [@johscheuer](https://github.com/johscheuer)) [SIG Cluster Lifecycle]
-- NONE ([#90944](https://github.com/kubernetes/kubernetes/pull/90944), [@Riaankl](https://github.com/Riaankl)) [SIG Testing]
+- NONE ([#90880](https://github.com/kubernetes/kubernetes/pull/90880), [@Riaankl](https://github.com/Riaankl)) [SIG Apps and Testing]
```

Comment:
This is suspicious. :)

Reply:
It seems both PRs came from the same feature branch:
https://github.com/kubernetes/kubernetes/pull/90880/commits
https://github.com/kubernetes/kubernetes/pull/90944/commits
They share like 15-20 commits, maybe that is why one replaced the other?

Reply:
Probably, might be possible that we do not excluded them because the release notes block (should start with ```release-note) is just a usual code block (starts with ```)

Reply:
Oh I see it does not have a note. I thought you meant that one was replacing the other.

Perhaps we should also start honoring the `release-note-none` label on the PR. Humans interpret "NONE" and such strings better than our `none|n/a|na` regexp

Reply:
Just wondering why our NONE regex does not exclude the PR. I thought that we already take the labels into account… 🤔

Reply:
Like you said it is a regular ``` block:
```text
"**What type of PR is this?**\r\n/kind cleanup\r\n\r\n**What this PR does / why we need it**:\r\nUpdates the test to rely on responses from `Patch` or `Update` instead of later `Get` or `List`\r\n\r\n**Which issue(s) this PR fixes**:\r\nFixes: #90881\r\nResolves #90957 \r\n\r\n [#88588 (comment)](https://github.com/kubernetes/kubernetes/pull/88588#issuecomment-607366847)\r\n\r\n**Special notes for your reviewer**:\r\nAttempts to fix flakiness.\r\n\r\n**Does this PR introduce a user-facing change?**:\r\n\r\n```\r\nNONE\r\n```\r\n\r\n**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:\r\n\r\n```\r\nNONE\r\n```\r\n\r\n/sig testing\r\n\r\n"
```
We do use the labels for the kinds and sigs but not for checking for inclusion. Since we have them already, it should be fairly cheap to skip the PR with `release-note-none`, I'll do that later today

Reply:
Sounds good, cool thank you! 🙂